templates/openstack/index.html

```diff
@@ -696,7 +696,7 @@ <h3 class="p-heading--4">Multiple products, single subscription</h3>
     <p class="u-sv3">Choose the plan that suits you the best:</p>
     {% with no_desktop="true" %}{% include "shared/pricing/_ua-for-infrastructure.html" %}{% endwith %}
     <p>
-      <a href="https://assets.ubuntu.com/v1/b0b42477-Ubuntu_Advantage_+Datasheet-2019.pdf" class="p-button">
+      <a href="https://assets.ubuntu.com/v1/83834ab4-B%20Ubuntu%20Pro%20DS%2028.10.22%20(1).pdf" class="p-button">
```

Comment:
```suggestion
      <a href="https://assets.ubuntu.com/v1/83834ab4-B%20Ubuntu%20Pro%20DS%2028.10.22.pdf" class="p-button">
```
Updated link on the copy doc so it doesn't have that weird (1) at the end